docs/src/pages/guide/how-to/02-adding-tools.mdx

```diff
@@ -96,6 +110,12 @@ async function main() {
 main();
 ```
 
+Run the code with:
+
+```bash copy
```

Comment:
should default to npx tsx

Reply:
tsx compiles to JS, and requires some configuration to get up and working.
bun comes shipped with a ts runner.

Reply:
i'm okay with bun for now but tsx is way more standard, eg 20x more npm downloads. Let's ship this but talk more offline